src/NLog/LayoutRenderers/Wrappers/CachedLayoutRendererWrapper.cs

```diff
@@ -47,7 +47,18 @@ namespace NLog.LayoutRenderers.Wrappers
     [ThreadAgnostic]
     public sealed class CachedLayoutRendererWrapper : WrapperLayoutRendererBase
     {
-        private string cachedValue;
+        /// <summary>
+        /// A value indicating when the cache is cleared.
+        /// </summary>
+        public enum ClearCacheOption 
```

Comment:
I agree with making this a `[Flags]` enum. I will change this and check to see if flags-enums work from the XML config.